cdap-ui/app/cdap/components/TextboxOnValium/index.js

```diff
@@ -32,8 +32,14 @@ export default class TextboxOnValium extends Component {
     }
   }
   updateTextValue(e) {
+    let textValue = e.target.value;
+
+    if (!this.props.allowSpace) {
+      textValue = textValue.trim();
```

Comment:
Spaces in the middle became impossible, since you can't enter space, unless the user is editing existing column